CONTRIBUTING.md

```diff
@@ -26,7 +26,7 @@ Thanks for contributing!
 
 * read [the Formula Cookbook](https://docs.brew.sh/Formula-Cookbook) or: `brew create $URL` and make edits
 * `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source foo`
-* `brew audit --new-formula foo`
```

Comment:
can you move this change to a separate PR?

Reply:
Moved to #177121